src/workflows-app/components/FilterableWorkflowTable.ts

```diff
@@ -151,6 +191,64 @@ const FilterableWorkflowTable = ({
     }
   };
 
+  const includeKey = useMemo(
+    () => [
+      'backendStatus',
+      'executionStatus',
+      'shardIndex',
+      'jobId',
+      'start',
+      'end',
+      'stderr',
+      'stdout',
+      'tes_stdout',
+      'tes_stderr',
+      'attempt',
+      'subWorkflowId', // needed for task type column
+    ],
+    []
+  );
+  const excludeKey = useMemo(() => [], []);
+
+  const getKeys = (keyValuePairs: {}): undefined => {
+    for (const key of Object.getOwnPropertyNames(keyValuePairs)) {
+      keyValuePairs[key];
+      delete Object.assign(keyValuePairs, {
+        [parseMethodString(key).variable]: keyValuePairs[key],
+      })[key];
+    }
+  };
```

Comment:
A lot of comments about this one... 

I find the name misleading. I would assume `getKeys(obj)` would return the keys of the passed object. However, it actually modifies the object in place. Generally, immutability makes code easier to reason about, and so it would be preferable to return a new object instead of modifying the passed object.

It _looks_ like this is transforming keys of the passed object by replacing `key` with `parseMethodString(key).variable`. If that's the case, I would suggest using Lodash's [`mapKeys`](https://lodash.com/docs/4.17.15#mapKeys) function to make this simpler and _much_ clearer.

Alternatively, you could implement a `mapKeys` in native JS with something like:
```js
const mapKeys = (obj, fn) => {
  return Object.fromEntries(
    Object.entries(obj).map(([key, value]) => [fn(key), value])
  );
};
```